src/jobs/sbt.yml

```diff
@@ -27,14 +27,26 @@ parameters:
     type: boolean
     description: "Whether to persist the workspace or not at the end of the job"
     default: false
-
+  updated_sbt:
```

Comment:
this parameter can be a `string` so that every user can tweak the sbt version to download.
The conditional should work as well checking against `null`.

Reply:
Can you take another look now @andreaTP?